go.mod

```diff
@@ -221,6 +220,7 @@ require (
 	gomodules.xyz/jsonpatch/v2 v2.2.0 // indirect
 	google.golang.org/appengine v1.6.7 // indirect
 	google.golang.org/genproto v0.0.0-20220107163113-42d7afdf6368 // indirect
+	google.golang.org/protobuf v1.27.1 // indirect
```

Comment:
Do we need genproto and protobuf packages?

Reply:
I tried to remove that and it complains so I guess it is a dependency with other modules.

Reply:
Probably because of grpc in the notebook read/write tests